pyro/contrib/tracking/dynamic_models.py

```diff
@@ -364,21 +367,22 @@ def process_noise_cov(self, dt=0.):
             stochastic integration (for use with EKF).
         '''
         if dt not in self._Q_cache:
-            # q: continuous-time process noise intensity with units
-            #   length^2/time^3 (m^2/s^3). Choose ``q`` so that changes in
-            #   velocity, over a sampling period ``dt``, are roughly
-            #   ``sqrt(q*dt)``.
-            q = self.sa2 * dt
-            d = self._dimension
-            dt2 = dt * dt
-            dt3 = dt2 * dt
-            Q = torch.zeros((d, d))
-            eye = eye_like(self.sa2, d//2)
-            Q[:d//2, :d//2] = dt3 * eye / 3.0
-            Q[:d//2, d//2:] = dt2 * eye / 2.0
-            Q[d//2:, :d//2] = dt2 * eye / 2.0
-            Q[d//2:, d//2:] = dt * eye
-            Q *= q
+            with torch.no_grad():
+                # q: continuous-time process noise intensity with
+                # units length^2/time^3 (m^2/s^3). Choose ``q`` so that
+                # changes in velocity, over a sampling period ``dt``,
+                # are roughly ``sqrt(q*dt)``.
+                q = self.sa2 * dt
```

Comment:
This should be outside of `torch.no_grad()`. Please re-run results and verify convergence after fixing grad errors. Also, doesn't this introduce an extra factor of `dt`?